docs/source/support/release-notes.rst

```diff
@@ -196,6 +196,8 @@ SQL Extractor
 --------------------
 
 - Making transaction lookups performant so we can handle such requests for large ledgers as well
+- We now color stale IDE diagnostics green to indicate that they are not current
```

Comment:
Looks like you've added this to a previous release, was that deliberate?

Also, nitpick - release notes bullets should end with a full stop.

Reply:
Good catch! Some of the joys of rebasing